test/e2e/manifest/testVPC/customize_networkconfig_updated.yaml

```diff
@@ -5,16 +5,11 @@ kind: VPCNetworkConfiguration
 metadata:
   name: selfdefinedconfig
 spec:
-  defaultGatewayPath: /infra/tier-0s/PLR
-  # nsx-operator-ci would replace '{edge-cluster-id}' with real edge-cluster-id of testbed
-  edgeClusterPath: /infra/sites/default/enforcement-points/default/edge-clusters/{edge-cluster-id}
-  defaultIPv4SubnetSize: 26
-  nsxtProject: /orgs/default/projects/nsx_operator_e2e_test
-  externalIPv4Blocks:
-    - /infra/ip-blocks/e2e_test_external_ip_blk
-  privateIPv4CIDRs:
+  defaultSubnetSize: 26
```

Comment:
please change defaultSubnetSize: to 32, since subnetszie must be power of 2.
so, 26 is invalid.